lib/ContainerManager.js

```diff
@@ -146,6 +151,31 @@ Server.prototype.configure = function(config, done) {
   done();
 };
 
+Server.prototype.jsonWebTokenParser = function(publicKey) {
+  var handleError = function(res) {
+    res.status(403);
+    res.json({error: 'Invalid JWT provided'});
+  };
+  return function(req, res, next) {
+    req.log.info(req.headers);
```

Comment:
Best to add a message to the log here, like `JWT Token Parser headers` and this should probably be logged at the `debug` level.